persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -72,6 +79,16 @@ class HmdaFileValidator(submissionId: String) extends HmdaPersistentActor with L
       val validationStarted = ValidationStarted(submissionId)
       publishEvent(validationStarted)
       events(parserPersistenceId)
+        .filter(x => x.isInstanceOf[TsParsed])
```

Comment:
Since there's only one TransmittalSheet per submission, should this be `find` instead of `filter`? (I haven't researched), but it seems like it would be less computationally intensive.